.github/workflows/eden_test.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Eden Test from lf-edge/eden
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "master"
+      - "[0-9]+.[0-9]+"
+      - "[0-9]+.[0-9]+-stable"
+    paths-ignore:
+      - 'docs/**'
+  pull_request_review:
+    types: [submitted]
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  call_test_wf_from_eden:
+    uses: lf-edge/eden/.github/workflows/test.yml@master
```

Comment:
How does eden know that it should run with EVE built for a given PR?
I mean [this](https://github.com/lf-edge/eve/blob/master/.github/workflows/eden.yml#L56-L87) and [this](https://github.com/lf-edge/eve/blob/master/.github/workflows/eden.yml#L97).

Reply:
Actually this is what I wanted to discuss. There's eden version and eve version as well. We need to test given EVE version with specific eden. I believe we need to add environment variable tag to specify EVE version and create versioning for eden (or at least versioning for test.yml action from eden) then when we bumping up eden version we are changing tag and EVE version will be chosen automatically.